packages/eslint-plugin/src/rules/no-unsafe-argument.ts

```diff
@@ -56,8 +56,7 @@ class FunctionSignature {
     let restType: RestType | null = null;
 
     const parameters = signature.getParameters();
-    for (let i = 0; i < parameters.length; i += 1) {
-      const param = parameters[i];
+    for (const [i, param] of parameters.entries()) {
```

Comment:
![Tim Gunn going 'eww' by sticking his tongue out and holding his hand up](https://github.com/user-attachments/assets/54397291-b134-4f1b-972c-7290e4024e7c)